source/win-main/DocumentTabs.vue

```diff
@@ -435,15 +454,16 @@ body.darwin {
 
   &.dark {
     div#tab-container {
-      border-bottom-color: rgb(11, 11, 11);
+      border-top-color: rgb(11, 11, 11);
 
       div[role="tab"] {
         color: rgb(233, 233, 233);
         background-color: rgb(22, 22, 22);
         border-color: rgb(22, 22, 22);
+        cursor: pointer;
```

Comment:
I'm pretty sure this shouldn't be done on any platform. The `pointer` cursor is (at least as far as I know, please correct me if I'm wrong) always intended for web-links, and never (!) for anything regarding the GUI, including the document tabs. Think about it: Is there a pointer-cursor on your browser's tabs?